src/i18n.js

```diff
@@ -0,0 +1,24 @@
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+
+Vue.use(VueI18n);
+
+function loadLocaleMessages() {
+  const locales = require.context('./locales', true, /[A-Za-z0-9-_,\s]+\.json$/i);
+  const messages = {};
+  locales.keys()
+    .forEach((key) => {
+      const matched = key.match(/([A-Za-z0-9-_]+)\./i);
+      if (matched && matched.length > 1) {
+        const locale = matched[1];
+        messages[locale] = locales(key);
+      }
+    });
+  return messages;
+}
+
+export default new VueI18n({
+  locale: process.env.VUE_APP_I18N_LOCALE || 'en',
```

Comment:
If you're loading the locales from an env variable, it's always going to be in one language for all users, isn't it? This should come from the browser information, and in the case of server rendering, there should be something in the request headers that indicate the client's preferred language

Reply:
This option only sets the default locale when building, the chosen locale has to be saved in a browser cookie or `localStorage` for permanence across sessions.

Reply:
Will incorporate this in an update! :+1: